log/gen.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package server holds code for core Trillian servers.
-package server
+// Package log holds code for core Trillian logs operation.
```

Comment:
Can you remove either comment this or the one in sequencer.go. Some linters don't like multiple package docs.